jscomp/bsc/rescript_compiler_main.ml

```diff
@@ -294,10 +292,6 @@ let buckle_script_flags : (string * Bsc_args.spec * string) array =
 
     (******************************************************************************)
 
-
-    "-bs-super-errors", unit_lazy Super_main.setup,
```

Comment:
Perhaps mention this as breaking change in the changelog.
I don't think there's any use for calling `bsc` directly, but just to be on the safe side.

Isn't there also something in `bsconfig`? Perhaps one could deprecate or remove that too.

Reply:
This is low risk. Remove is OK. No need to deprecate.

Reply:
https://github.com/rescript-lang/rescript-compiler/pull/6199/commits/2c24e15fcfa722e2cc5065126c95cf14543955f6